pydicom/tests/test_gdcm_pixel_data.py

```diff
@@ -63,18 +71,27 @@
 save_dir = os.getcwd()
 
 
-@pytest.mark.skipif(have_gdcm_handler, reason=gdcm_present_message)
 class GDCM_JPEG_LS_Tests_no_gdcm(unittest.TestCase):
     def setUp(self):
-        self.jpeg_ls_lossless = read_file(jpeg_ls_lossless_name)
+        self.utf8_filename = os.path.join(tempfile.gettempdir(), "ДИКОМ.dcm")
+        shutil.copyfile(jpeg_ls_lossless_name, self.utf8_filename)
+        try:
+            self.jpeg_ls_lossless = read_file(
+                unicode(self.utf8_filename,
+                        encoding=sys.getfilesystemencoding()))
+        except NameError:
```

Comment:
I think it is much better in Python 3, but still can be confusing with the default encoding, the file system encoding and explicit encoding used in different places. Usually you are fine if you stick to explicit utf8 encoding, but especially under Windows, where the default encoding is something like latin1, and file names are encoded as utf16, this can be a pain.